recipe/meta.yaml

```
@@ -22,22 +22,25 @@ requirements:
     - pip
```

```suggestion
    - pip
    - jupyter-packaging >=0.12,<1
```
Maybe the `jupyter-packaging` requirement should be added in `host` section, like here https://github.com/conda-forge/jupyterlab-feedstock/blob/91e973dc9e99afa7b696eb0e9b1bf8d2316747aa/recipe/meta.yaml#L23